doc/how-to/remove_machine.md

```diff
@@ -7,8 +7,8 @@ If you want to remove a machine from the MicroCloud cluster after the initialisa
 
 Before removing the machine, ensure that there are no LXD instances or storage volumes located on the machine, and that there are no MicroCeph OSDs located on the machine.
 
-See [how to remove instances](https://documentation.ubuntu.com/lxd/en/latest/howto/instances_manage/#delete-an-instance) in the LXD documentation.
-See [how to remove OSDs](https://canonical-microceph.readthedocs-hosted.com/en/latest/how-to/remove-disk/) in the MicroCeph documentation.
+See {ref}`how to remove instances <lxd:instances-manage>` in the LXD documentation.
```

Comment:
This one goes to the page now instead of the anchor. Let me add an anchor to the LXD docs. (Shouldn't block merging though, we can update this later ;) ).